### DISCUSSION TITLE
keyball39 右手トラックボール側の2番LEDのパッドをはがしてしまった　

### DISCUSSION THREAD

**Discussion by Yoichi-Maribao** [Yowkees/keyball#355] [Q&A]
表題の件について、他のディスカッションを読んで2を飛ばして1と3以降を光らすことはできそうなんですが
2を復活させるにはどこに繋げば良いかわかりませんでした。

剥がれてしまった箇所は緑のポッチが書き込んである、右側二つと左下一つです。

お手隙の際にご教示いただけますと幸いです。
![IMG_5790](https://github.com/Yowkees/keyball/assets/64699192/1efa2597-6fbe-4e43-b366-b9f7f38a4a0e)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

下記画像の同色どうし３か所をエナメル線やゴム被覆導線で接続していただくと、LED2を復活させることができます。
LED1が光っていて見にくいですが、右上の端子になります。

以上、お手数ですがご対応お願いいたします。
![KB39_LED2fix](https://github.com/Yowkees/keyball/assets/84209772/1f7d3ea1-db0a-4e0a-b4dd-49b3488e0db4)

**Reply by Yoichi-Maribao** [reply]
無事復活させることができました！ありがとうございます！！

**Comment by Yowkees** [comment]
無事に直ったようでよかったです。
今後ともどうぞよろしくお願いいたします。
